### PR TITLE
[devtools-translate] Portuguese - What's New in DevTools (Chrome 102)

### LINKED ISSUE #2803: [devtools-translate] Portuguese - What's New in DevTools (Chrome 102)
Post: https://developer.chrome.com/blog/new-in-devtools-102/
Translated by: May 17

Translator: @khaosdoctor 
Reviewers: @alvarocamillont

### PR DESCRIPTION
**Pull request by khaosdoctor**
Fixes #2803

Changes proposed in this pull request:

- Translation to Brazilian PT

### CODE REVIEW

File: site/pt/blog/new-in-devtools-102/index.md [hunk 1]
```diff
@@ -28,126 +27,156 @@ draft: true
...
-Chromium issue: [1295750](https://crbug.com/1295750)
+Issue relacionada: [1295750](https://crbug.com/1295750)
 
 
 <!-- ### Commit live expression with Enter {: #live-expression } -->
-
+### Enter para enviar uma *live expression*  {: #live-expression }
 <!-- Once you finish typing a [live expression](/blog/new-in-devtools-70/#watch), you can click `Enter` to commit it. Previously, hitting Enter resulted in adding new lines. This is inconsistent with other parts of the DevTools.  -->
-
+Quando você terminar de digitar uma [live expression](/blog/new-in-devtools-70/#watch), você pode apertar `Enter` para enviá-la. Anteriormente, apertar `Enter` resultava na adição de novas linhas. Isso é inconsistente com outras partes do DevTools.
```
Comment (alvarocamillont) [inline site/pt/blog/new-in-devtools-102/index.md:124]: Quando você terminar de digitar uma [live expression](/blog/new-in-devtools-70/#watch), você pode apertar `Enter` para enviá-la. Anteriormente, apertar `Enter` resultava na adição de novas linhas. Isso era inconsistente com outras partes do DevTools.

File: site/pt/blog/new-in-devtools-102/index.md [hunk 2]
```diff
@@ -28,126 +27,156 @@ draft: true
...
 
-Chromium issue: [1260744](https://crbug.com/1260744)
+Issue relacionada: [1260744](https://crbug.com/1260744)
 
 
 <!-- ## Cancel user flow recording at the start {: #recorder } -->
-
+## Cancelamento de gravações de fluxo de usuário no início {: #recorder }
 <!-- You can cancel the recording during the start of user flow recording. Previously, there was no option to cancel the recording. -->
+Você pode cancelar a gravação durante o início de uma gravação de fluxo de usuário. Anteriormente não havia essa opção.
```
Comment (alvarocamillont) [inline site/pt/blog/new-in-devtools-102/index.md:138]: Você pode cancelar a gravação durante o início do registro do fluxo de usuário. Anteriormente não havia essa opção.

**Review by alvarocamillont — COMMENTED** [review]
Some small adjustments :)

**Comment by jecfish** [comment]
Thanks Lucas! Over to @andreban for review.

File: site/pt/blog/new-in-devtools-102/index.md (hunk 2 above)
Comment (khaosdoctor) [reply site/pt/blog/new-in-devtools-102/index.md:138]:
```suggestion
Você pode cancelar a gravação durante o início de um registro de fluxo de usuário. Anteriormente não havia essa opção.
```

File: site/pt/blog/new-in-devtools-102/index.md (hunk 1 above)
Comment (khaosdoctor) [reply site/pt/blog/new-in-devtools-102/index.md:124]:
```suggestion
Quando você terminar de digitar uma [live expression](/blog/new-in-devtools-70/#watch), você pode apertar `Enter` para enviá-la. Anteriormente, apertar `Enter` resultava na adição de novas linhas. Isso era inconsistente com outras partes do DevTools.
```

**Review by andreban — APPROVED** [review]
Approved, as long as we can check the below:

When the article mentions text rendered in the browser UI, some of the text is in english. I don't have Chrome setup in Portuguese, so I wasn't able to double check if those are mixed in the UI itself or just in the article. Would you mind double checking?

File: site/pt/blog/new-in-devtools-102/index.md [hunk 3]
```diff
@@ -156,41 +185,50 @@ Chromium issue: [1024156](https://crbug.com/1024156)
...
-Chromium issue: [1268754](https://crbug.com/1268754)
+Issue relacionada: [1268754](https://crbug.com/1268754)
 
 
 <!-- ## [Experimental] Picking color outside of browser {: #color-picker } -->
+## [Experimental] Escolhendo cores fora do browser {: #color-picker }
 
 {% Aside %}
 <!-- To enable the experiment, check **Enable color picking outside the browser window** under **Settings** > **Experiments**. -->
+Para ativar esse experimento, ative a flag **Enable color picking outside the browser window** no painel **Configurações** > **Experimentos**
```
[the remark this answers is not in the document]
Comment (khaosdoctor) [reply site/pt/blog/new-in-devtools-102/index.md:218]: Yeah, this is something I've been questioning myself; my browser is in English fully so I don't have it translated to Portuguese. I tested with my work's laptop and it seems correct, the flags are in English whilst the menu is translated.

File: site/pt/blog/new-in-devtools-102/index.md [hunk 4]
```diff
@@ -156,41 +185,50 @@ Chromium issue: [1024156](https://crbug.com/1024156)
 
 
 <!-- ## [Experimental] Copy CSS changes {: #copy } -->
+## [Experimental] Copie mudanças de CSS {: #copy }
 
 {% Aside %}
 <!-- To enable the experiment, check **Sync CSS changes in the Styles pane** under **Settings** > **Experiments**. -->
+Para ativar esse experimento, ative a flag **Sync CSS changes in the Styles pane** no painel **Configurações** > **Experimentos**
```
[the remark this answers is not in the document]
Comment (khaosdoctor) [reply site/pt/blog/new-in-devtools-102/index.md:192]: Yeah, this is something I've been questioning myself; my browser is in English fully so I don't have it translated to Portuguese. I tested with my work's laptop and it seems correct, the flags are in English whilst the menu is translated.

File: site/pt/blog/new-in-devtools-102/index.md [hunk 5]
```diff
@@ -28,126 +27,156 @@ draft: true
...
 - The **Styles** pane now properly shows the overridden `@support` rules as strikethrough. Previously, the rules weren’t strikethrough. ([1298025](https://crbug.com/1298025))
 - Fixed the CSS formatting logic in the **Sources** panel that caused multiple blank lines when editing CSS. ([1309588](https://crbug.com/1309588))
 - Cap the **Expand recursively** option of an object in the **Console** to maximum 100 so it does not go on forever for circular objects. ([1272450](https://crbug.com/1272450)) -->
-
+- O painel **Propriedades** agora mostra os acessores de cada propriedade com seus valores por padrão. Ele estava escondido por um erro anteriormente. ([1309087](https://crbug.com/1309087))
+- O painel **Estilos** agora mostra as regras sobrescritas de `@support` de forma correta como riscadas. Anteriormente elas não estavam sendo exibidas nesse estilo. ([1298025](https://crbug.com/1298025))
+- Logica de formação de CSS no painel de **Sources** que causava múltiplas linhas em branco quando editando CSS foi corrigida. ([1309588](https://crbug.com/1309588))
```
[the remark this answers is not in the document]
Comment (khaosdoctor) [reply site/pt/blog/new-in-devtools-102/index.md:178]: It is, I'm applying the suggestion below
```suggestion
- Logica de formação de CSS no painel de **Fontes** que causava múltiplas linhas em branco quando editando CSS foi corrigida. ([1309588](https://crbug.com/1309588))
```

**Review by andreban — APPROVED** [review]
LGTM

**Comment by jecfish** [comment]
Thanks all!

**Pull request merged**